src/clj/game/cards/resources.clj

```diff
@@ -614,7 +606,13 @@
       :abilities [ability]
       :move-zone heap-event
       :events {:runner-turn-begins ability
-               :runner-turn-ends nil}})
+               :runner-turn-ends nil
+               :counter-added {:req (req (not (pos? (get-counters card :credit))))
```

Comment:
Doesn't this need a `(req (same-card? card target)`?

Reply:
Yep! Good catch.